playwright/tests/organize/people/person/tags.spec.ts

```diff
@@ -317,7 +317,7 @@ test.describe('Person Profile Page Tags', () => {
         ActivistTag.title
       );
 
-      await page.click('data-testid=submit-button');
```

Comment:
Thanks for renaming this!

Reply:
I appreciate you saying that! I had given it a test ID in my branch and had to pick one of the two when I got a merge conflict, so I went with mine. 😊